pkg/ui/NodeInputCheckbox.tsx

```diff
@@ -14,8 +14,8 @@ export function NodeInputCheckbox<T>({
     <>
       <Checkbox
         name={attributes.name}
-        defaultChecked={attributes.value === true}
-        onChange={(e) => setValue(e.target.checked)}
+        defaultChecked={Boolean(attributes.value)}
+        onChange={(e) => setValue(Boolean(e.target.checked))}
```

Comment:
You should be careful as the `Boolean` function uses [coercion to convert to bool](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean#boolean_coercion) which might not be what you want.
You should think about what the possible values are and what constitutes `true` and `false` (eg. maybe the string `"no"` should be converted to `false`?).

Reply:
Yes you are correct, I forgot to remove these since these tests were failing in Kratos due to the old `@ory/themes` library adding a string 'yes' or 'no' value to the checkbox instead of using a native `true` or `false`